Use the right separator in file path.

martinvehmas opened the pull request:
On Windows the directory path can include backslash () and thats what
path.dirname returns. The file path is then added with a slash (/).

This will result in paths like "**services\marketing/email.html**"

This commit replace backslash with slash in the directory path when its
concatenated with the filename.

martinvehmas committed:
Replace backslash with slash in directory path.

On Windows the directory path can include backslash (\) and thats what
path.dirname returns. The file path is then added with a slash (/).

This will result in paths like "services\marketing/email.html"

This commit replace backslash with slash in the directory path when its
concatenated with the filename.

RobLoach commented:
Could [path.join](https://nodejs.org/api/path.html#path_path_join_path1_path2) help here?

martinvehmas committed:
Join the path as a file path with path.join

The path is not a url so the hard coded separator / should not be used.

Its still file paths even if its .html files.

path.join is used to join the dir and the file.

martinvehmas commented:
I thought the output should be a URL, but of course its still a file path.
path.join should be the right way to do it to get the right separator.
I updated the Pull request.

RobLoach commented:
Nicely done, LGTM :+1:

paazmaya commented:
Any chance of merging? @RobLoach maybe?

RobLoach commented:
I don't have maintainership here. @ianstormtaylor perhaps?

woodyrew commented:
Sorry for the delay, thank you.
